deploy/file_list

```diff
@@ -1,8 +1,10 @@
-Assets/Plugins/Android/SoomlaAndroidCore.jar
```

Comment:
@EugeneButusov we can now simply write:
Assets/Plugins/Android/SOOMLA
Assets/Plugins/iOS/SOOMLA

instead of writing each file